### PR TITLE
Update minimap2 to install the man page

### PR DESCRIPTION
**Pull request by nh13**
Describe your pull request here

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

File: recipes/minimap2/meta.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ source:
   sha256: f4c8c3459c7b87e9de6cbed7de019b48d9337c2e46b87ba81b9f72d889420b3c
 
 build:
-  number: 0
+  number: 1
```
Comment (nh13) [inline recipes/minimap2/meta.yaml:12]: Why is this failing lint? The build number is incremented relative to master: https://github.com/bioconda/bioconda-recipes/blob/af04e627ab11092e31545d965dbca42704b2ca80/recipes/minimap2/meta.yaml#L12

Comment (bgruening) [reply recipes/minimap2/meta.yaml]: https://anaconda.org/bioconda/minimap2/files

We have rebuilt a new version on bulk I think in the meantime - which is not yet merged back to main.

**Comment by nh13** [comment]
@BiocondaBot please add label

File: recipes/minimap2/build.sh [hunk 2]
```diff
@@ -22,3 +22,9 @@ cp minimap2 misc/paftools.js ${PREFIX}/bin
 cp sdust ${PREFIX}/bin
 cp libminimap2.a ${PREFIX}/lib
 cp *.h ${PREFIX}/include
+
+if [ ! -d $PREFIX/share/man/man1 ] ; then
```
Comment (martin-g) [inline recipes/minimap2/build.sh:26]: There is no need of the check. 
mkdir -p would not fail if the folder already exists

File: recipes/minimap2/build.sh [hunk 3]
```diff
@@ -22,3 +22,9 @@ cp minimap2 misc/paftools.js ${PREFIX}/bin
...
+  mkdir -p $PREFIX/share/man/man1
+fi
```
Comment (nh13) [inline recipes/minimap2/build.sh:28]:
```suggestion
mkdir -p $PREFIX/share/man/man1
```

File: recipes/minimap2/build.sh [hunk 4]
```diff
@@ -22,3 +22,7 @@ cp minimap2 misc/paftools.js ${PREFIX}/bin
 cp sdust ${PREFIX}/bin
 cp libminimap2.a ${PREFIX}/lib
 cp *.h ${PREFIX}/include
+
+mkdir -p $PREFIX/share/man/man1
+
+cp minimap2.1 $PREFIX/share/man/man1
```
Comment (nh13) [inline recipes/minimap2/build.sh:28]:
```suggestion
mkdir -p $PREFIX/share/man/man1
cp minimap2.1 $PREFIX/share/man/man1
```

**Comment by nh13** [comment]
I think this is ready for a final review

**Review by martin-g — APPROVED** [review]

**Pull request merged**